vignettes/classes-objects.Rmd

```diff
@@ -305,24 +306,24 @@ hadley <- Person(firstName = "Hadley")
 
 hadley <- Person(first_name = "Hadley") # no warning
 
-hadley@firstName                       
+hadley@firstName
 
-hadley@firstName <- "John"              
+hadley@firstName <- "John"
 
 hadley@first_name  # no warning
 ```
 
-
 #### Required properties
 
 You can make a property required by the constructor either by:
 
-- relying on the validator to error with the default value, or by
-- setting the property default to a quoted error call.
+-   relying on the validator to error with the default value, or by
+-   setting the property default to a quoted error call.
 
 ```{r}
 Person <- new_class("Person", properties = list(
- name = new_property(class_character, validator = function(value) {
+ name = new_property(class_character, 
```

Comment:
If you're going to spread over multiple lines, I think `class_character` should be on its own line too.